sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Customizations/DatabasesOperations.cs

```diff
@@ -61,5 +63,50 @@ internal partial class DatabasesOperations
                 customHeaders,
                 cancellationToken).ConfigureAwait(false);
         }
+
```

Comment:
This looks good, you would need similar customizations in the partial interface for IDatabasesOperations and the ListByServerAsync method in the DatabasesOperationsExtensions partial class [although, since this is an extension method, it doden't need to reside in the same static class]

Reply:
ok I understand now. Made the changes to add the interface in IDatabasesOperations, and add ListByServerAsync without skip token in the DatabasesOperationsExtensions class